__tests__/api.test.js

```diff
@@ -106,3 +106,49 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    test("should return an array of comments by article_id in url with a status of 200", () => {
+      return request(app)
+        .get("/api/articles/3/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBeGreaterThan(0);
+          comments.forEach((comment) => {
+            expect(comment).toEqual({
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              article_id: expect.any(Number),
```

Comment:
You are asking for the comments for article 3. Let's assert that you are expecting the article_id is 3 instead of `expect.any()`